moped-database/migrations/1665693611607_add_component_geometry_tables/up.sql

```diff
@@ -0,0 +1,201 @@
+    symbol integer default null,
+    source_layer character varying,
+    geography geography('MULTILINESTRING') default null
+    ) inherits (features);
+
+create table feature_drawn_points (
+    geography geography('MULTILINESTRING') default null
+    ) inherits (features);
+
+create table feature_drawn_lines (
+    geography geography('MULTILINESTRING') default null
+    ) inherits (features);
+
+create table component_feature_map (
+    id serial primary key,
+    component_id integer references moped_proj_components(project_component_id),
+    feature_id integer default null
+    );
+
+create view uniform_features as (
+        select id, 'feature_signals' as table, name, json_build_object(
+            'signal_id', signal_id,
+            'knack_id', knack_id,
+            'project_extent_id', project_extent_id,
+            'location_name', location_name,
+            'render_type', render_type,
+            'signal_type', signal_type,
+            'source_layer', source_layer
+            ) as attributes, geography
+        FROM feature_signals
+    union all
+        select id, 'feature_street_segments' as table, name, json_build_object(
+            'knack_id', knack_id,
+            'ctn_segment_id', ctn_segment_id,
+            'project_extent_id', project_extent_id,
+            'from_address_min', from_address_min,
+            'render_type', render_type,
+            'to_address_max', to_address_max,
+            'full_street_name', full_street_name,
+            'line_type', line_type,
+            'symbol', symbol,
+            'source_layer', source_layer
+            ) as attributes, geography
+        FROM feature_street_segments
+    union all
+        select id, 'feature_intersections' as table, name, json_build_object(
+            'intersection_id', intersection_id,
+            'project_extent_id', project_extent_id,
+            'source_layer', source_layer,
+            'render_type', render_type
+            ) as attributes, geography
+        FROM feature_intersections
+    union all
+        select id, 'feature_drawn_points' as table, name, null as attributes, geography from feature_drawn_points
+    union all
+        select id, 'feature_drawn_lines' as table, name, null as attributes, geography from feature_drawn_lines
+);
+
+create view project_geography as (
+    select 
+      moped_project.project_id as project_id,
+      uniform_features.id as feature_id,
+      moped_components.component_id as component_archtype_id,
+      moped_proj_components.project_component_id as component_id,
+      moped_proj_components.is_deleted,
+      moped_project.project_name as project_name,
+      feature_layers.internal_table as table,
+      feature_layers.reference_layer_primary_key_column as original_fk,
+      moped_proj_components.name as component_name,
+      uniform_features.name as feature_name,
+      uniform_features.attributes as attributes,
+      uniform_features.geography as geography
```

Comment:
What particular columns we should include in this view will likely change to reflect the needs and convenience of the work being done to the mapping front end systems.

Reply:
We circled up to talk about the mapping systems today, and we observed that this view is really more of a example of what we can do, and not so much intended to be the prescribed form that the data should be made available via the graphql-engine. 

We agreed to spin-off a new issue to track the refinement of this view and creation of others as needed based on what @mddilley finds to be the best endpoint shape for his work on the frontend.

Reply:
👍 will make that issue

Reply:
Here's that issue https://github.com/cityofaustin/atd-data-tech/issues/10606